src/components/QuestionerButton/QuestionerButton.css

```diff
@@ -0,0 +1,44 @@
+.questioner-button {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+  justify-content: center;
+}
+
+.questioner-button-back {
```

Comment:
It looks like `.questioner-button-back` and `.questioner-button-next` share a lot of styles. How about separating the class names to `.questioner-button`, `.back`, `.next` instead?